### DISCUSSION TITLE
Increase IO performance

### DISCUSSION THREAD

**Discussion by samhatem** [planetscale/discussion#236] [Q&A]
What are some recommendations to increase read + write performance on planet scale?

For context, we appear to be hitting a DB bottleneck doing some load testing. We're submitting 100s of reads per second and seeing network latency in the range of 2-11s which is significantly higher than anticipated. It's a python microservice making the requests and we're using [sqlalchemy](https://www.sqlalchemy.org/) as an ORM.

**Comment by orware** [comment]
@samhatem,

Since we are currently working with you on these questions via your ticket submitted into our support system, we'll continue working there for easier communication, but we'll come back here with additional details / resolution at the end of the process if there is something that will be useful for others to use as a reference.

**Reply by orware** [reply]
@samhatem,

I wanted to close the loop on this one since you were able to self-solve the performance issue you were experiencing. 

Since we didn't receive many extra details about the solution in the ticket, other than it not being PlanetScale related in the end, I cannot provide extra details here for other users that might be helpful, but if you or one of the others on your team are comfortable with adding some of those details here, I'm certain other users will find the tips useful!

**Comment by pedro-lucca** [comment]
👀
